src/libs/ajax/data-table-providers/WdsDataTableProvider.ts

```diff
@@ -317,4 +317,22 @@ export class WdsDataTableProvider implements DataTableProvider {
       uploadParams.file
     );
   };
+
+  importTdr = (workspaceId: string, snapshotId: string): Promise<Response> => {
+    if (!this.proxyUrl) return Promise.reject('Proxy Url not loaded');
+    setTimeout(() => {
+      if (
+        notificationStore.get().length === 0 ||
+        !notificationStore
+          .get()
+          .some((notif: { id: string }) => ['tdr-imports', 'tdr-import_success'].includes(notif.id))
+      ) {
+        notifyDataImportProgress(
+          'tdr-import',
+          'Your data will show up under Tables once import is complete.  Please refresh the page to see your changes.'
+        );
+      }
+    }, 1000);
+    return Ajax().WorkspaceData.importTdr(this.proxyUrl, workspaceId, snapshotId);
```

Comment:
As the import is not yet async, it's not easy to notify when the import actually completes.  Since we're not yet doing a full import however, in practice it seems to happen quickly enough that a refresh right away will show the new table.